### PR TITLE
JSON Update + ResourceCard reorder

### PR DESCRIPTION
**Pull request by RichKummer**
### Related Ticket(s)

[#9096](https://app.zenhub.com/workspaces/carbon-for-ibmcom-5d449f3642eb1962336cbe52/issues/carbon-design-system/carbon-for-ibm-dotcom/9096)

### Description

Added `React wrapper` ResourceCards to remaining components:

**Relevant component pages:**

- [x] Back to top
- [x] Callout quote
- [x] Callout with media
- [x] Card group
- [x] Card in card
- [x] CTA > Card link
- [x] Card section carousel
- [x] Card 
- [x] Carousel
- [x] Content block simple
- [x] Content block horizontal
- [x] Content block media
- [x] Content block
- [x] Content group banner
- [x] Content group
- [x] Content item horizontal
- [x] Content item
- [x] Content section
- [x] CTA section
- [x] Expressive modal
- [x] Feature card
- [x] Feature section
- [x] Filter panel
- [x] Image (& Image with caption)
- [x] Lead space block
- [x] Lead space search
- [x] Lead space
- [x] Leaving IBM
- [x] Logo grid
- [x] Masthead
- [x] Pictogram item
- [x] Quote
- [x] Table of contents
- [x] Tabs extended media
- [x] Tabs extended
- [x] Tag link
- [x] Universal banner
- [x] Video player
* Card section offset (NO PAGE)
* Content block mixed (NO PAGE)
* Search with typeahead (NO PAGE)
* Tag group (NO PAGE)

### Changelog

**New**

- N/A

**Changed**

- Changed ordering of ResourceCard so `React wrapper` follows `React`
- Added `React wrapper` data to JSON for relevant components

**Removed**

- N/A

### CODE REVIEW

File: src/data/components.json [hunk 1]
```diff
@@ -1496,7 +1519,8 @@
       "description": "The table of contents component allows users to quickly navigate through long pages by providing jump links to different sections of the content within a single page.",
       "storybook": {
         "webcomponents": "http://www.ibm.com/standards/carbon/web-components/?path=/story/components-table-of-contents--default",
-        "react": "http://www.ibm.com/standards/carbon/react/?path=/story/components-table-of-contents--manually-define-menu-items"
+        "react": "http://www.ibm.com/standards/carbon/react/?path=/story/components-table-of-contents--manually-define-menu-items",
```
Comment (kennylam) [inline src/data/components.json:1522]:
```suggestion
        "react": "https://www.ibm.com/standards/carbon/react/?path=/story/components-table-of-contents--default",
```

File: src/data/components.json [hunk 2]
```diff
@@ -1415,7 +1436,7 @@
       "description": "Lead space search provides a fast route to information discovery by including a prominent search option in the lead space. As the search is always positioned at the top, it is discoverable by users the moment they land on the page.",
       "storybook": {
         "webcomponents": "http://www.ibm.com/standards/carbon/web-components",
```
Comment (kennylam) [inline src/data/components.json:1438]:
```suggestion
        "webcomponents": "https://www.ibm.com/standards/carbon/web-components/?path=/story/components-lead-space-search--default",
```

File: src/data/components.json [hunk 3]
```diff
@@ -1361,7 +1380,8 @@
       "description": "Lead space sets the context and also helps users understand what type of content they are going to find on the page. It is always positioned at the top.",
       "storybook": {
         "webcomponents": "http://www.ibm.com/standards/carbon/web-components/?path=/story/components-leadspace--default-with-no-image",
```
Comment (kennylam) [inline src/data/components.json:1382]:
```suggestion
        "webcomponents": "https://www.ibm.com/standards/carbon/web-components/?path=/story/components-lead-space--super",
```

File: src/data/components.json [hunk 4]
```diff
@@ -1226,7 +1244,8 @@
       "description": "Content section is a high-level content component that acts as the main content unit to build web page experiences for IBM.com. It encapsulates other content components (one or multiple Content blocks and/or Content groups) and brings them together, providing flexibility to create many different solutions for users.",
       "storybook": {
         "webcomponents": "http://www.ibm.com/standards/carbon/web-components/",
-        "react": "http://www.ibm.com/standards/carbon/react/"
+        "react": "http://www.ibm.com/standards/carbon/react/",
+        "reactwrapper": "https://www.ibm.com/standards/carbon/web-components/react/?path=/story/components-content-section--default"
```
Comment (kennylam) [inline src/data/components.json:1248]:
```suggestion
        "react": "https://www.ibm.com/standards/carbon/react/?path=/story/components-content-section--default",
        "reactwrapper": "https://www.ibm.com/standards/carbon/web-components/react/?path=/story/components-content-section--default"
```

File: src/data/components.json [hunk 5]
```diff
@@ -1226,7 +1244,8 @@
       "description": "Content section is a high-level content component that acts as the main content unit to build web page experiences for IBM.com. It encapsulates other content components (one or multiple Content blocks and/or Content groups) and brings them together, providing flexibility to create many different solutions for users.",
       "storybook": {
         "webcomponents": "http://www.ibm.com/standards/carbon/web-components/",
```
Comment (kennylam) [inline src/data/components.json:1246]:
```suggestion
        "webcomponents": "https://www.ibm.com/standards/carbon/web-components/?path=/story/components-content-section--default",
```

File: src/data/components.json [hunk 6]
```diff
@@ -956,7 +969,8 @@
       "description": "Content block media is used to present information with images in a group setting.",
       "storybook": {
         "webcomponents": "http://www.ibm.com/standards/carbon/web-components/?path=/story/components-content-block-media--default",
-        "react": "http://www.ibm.com/standards/carbon/react/?path=/story/components-content-block-media--default"
+        "react": "http://www.ibm.com/standards/carbon/react/?path=/story/components-content-block-media--default",
+        "reactwrapper": "https://www.ibm.com/standards/carbon/web-components/react/?path=/story/components-content-block-horizontal--default"
```
Comment (kennylam) [inline src/data/components.json:973]:
```suggestion
        "reactwrapper": "https://www.ibm.com/standards/carbon/web-components/react/?path=/story/components-content-block-media--default"
```

File: src/data/components.json [hunk 7]
```diff
@@ -929,7 +941,8 @@
       "description": "Content block horizontal is used to highlight a group of more important pieces of content, such as products or solutions, with more visual weight.",
       "storybook": {
         "webcomponents": "https://www.ibm.com/standards/carbon/web-components/?path=/story/components-content-group-horizontal--default",
```
Comment (kennylam) [inline src/data/components.json:943]:
```suggestion
        "webcomponents": "https://www.ibm.com/standards/carbon/web-components/?path=/story/components-content-block-horizontal--default",
```

File: src/data/components.json [hunk 8]
```diff
@@ -929,7 +941,8 @@
...
-        "react": "https://www.ibm.com/standards/carbon/react/?path=/story/components-content-group-horizontal--default"
+        "react": "https://www.ibm.com/standards/carbon/react/?path=/story/components-content-group-horizontal--default",
```
Comment (kennylam) [inline src/data/components.json:944]:
```suggestion
        "react": "https://www.ibm.com/standards/carbon/react/?path=/story/components-content-block-horizontal--default",
```

File: src/data/components.json [hunk 9]
```diff
@@ -223,7 +226,8 @@
       "description": "Feature card is used to emphasize a piece of information with a card on the right-hand side and an image on the left-hand side.",
```
Comment (kennylam) [inline src/data/components.json:226]: Should there be a page for `feature-card`? It's not in the components list.

File: src/data/components.json [hunk 10]
```diff
@@ -196,7 +198,8 @@
       "description": "Expressive modal is based on the Carbon modal component with slight styling updates to increase the padding and uses fixed buttons.",
       "storybook": {
         "webcomponents": "http://www.ibm.com/standards/carbon/web-components/?path=/story/components-modal--default",
```
Comment (kennylam) [inline src/data/components.json:200]:
```suggestion
        "webcomponents": "https://www.ibm.com/standards/carbon/web-components/?path=/story/components-expressive-modal--default",
```

File: src/data/components.json [hunk 11]
```diff
@@ -114,7 +115,8 @@
       "description": "Card link is a CTA styled like a card that acts as the main CTA for a section of content, it typically appears at the end of a block-pattern.",
```
Comment (kennylam) [inline src/data/components.json:115]: Should there be a page for `card-link`? It's not in the components list.

File: src/data/components.json [hunk 12]
```diff
@@ -87,7 +88,7 @@
       "description": "Card in card offers a large media moment, while keeping harmony in a card group.",
```
Comment (kennylam) [inline src/data/components.json:88]: Should there be a page for `card-in-card`? It's not in the components list.

**Review by kennylam — COMMENTED** [review]
@RichKummer Just a few URL updates and the rest looks good!

File: src/data/components.json (hunk 9 above)
Comment (RichKummer) [reply src/data/components.json:226]: The `feature-card`, `card-link`, and `card-in-card` all appear on the `Card` page as their own sections. Each should have a ResourceCard, but I'll double check to make sure

**Comment by RichKummer** [comment]
@kennylam fixed the referenced URLs, and confirmed that the ResourceCards are right on the `Card`-related pages

File: src/data/components.json (hunk 3 above)
Comment (oliviaflory) [inline src/data/components.json]:
```suggestion
        "webcomponents": "https://www.ibm.com/standards/carbon/web-components/?path=/story/components-lead-space--super",
```

File: src/data/components.json (hunk 2 above)
Comment (oliviaflory) [inline src/data/components.json]:
```suggestion
        "webcomponents": "https://www.ibm.com/standards/carbon/web-components/?path=/story/components-lead-space-search--default",
```

File: src/data/components.json (hunk 1 above)
Comment (oliviaflory) [inline src/data/components.json]:
```suggestion
        "react": "https://www.ibm.com/standards/carbon/react/?path=/story/components-table-of-contents--default",
```

File: src/data/components.json [hunk 13]
```diff
@@ -1226,7 +1244,8 @@
       "description": "Content section is a high-level content component that acts as the main content unit to build web page experiences for IBM.com. It encapsulates other content components (one or multiple Content blocks and/or Content groups) and brings them together, providing flexibility to create many different solutions for users.",
       "storybook": {
         "webcomponents": "http://www.ibm.com/standards/carbon/web-components/",
-        "react": "http://www.ibm.com/standards/carbon/react/"
+        "react": "http://www.ibm.com/standards/carbon/react/",
```
[the remark this answers is not in the document]
Comment (kennylam) [reply src/data/components.json]: @oliviaflory I think I got that one already https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/pull/1499#discussion_r933386643

File: src/data/components.json (hunk 8 above)
[the remark this answers is not in the document]
Comment (RichKummer) [reply src/data/components.json]: Looks like the `Content block` links were all pointing to `group`, fixing that in the next commit

**Review by kennylam — APPROVED** [review]
LGTM!

**Pull request merged**